project/src/components/app/app.tsx

```diff
@@ -9,39 +13,45 @@ import Property from '../property/property';
 import PrivateRoute from '../private-route/private-route';
 
 import { Offer } from '../../types/offer';
-import { Review } from '../../types/review';
+import { Actions } from '../../types/action';
+import { setOfferList } from '../../store/actions';
 
-type AppProps = {
-  cardsAmount: number;
-  offers: Offer[];
-  reviews: Review[];
-}
+import { offers as mockOffers } from '../../mocks/offers';
+import { reviews } from '../../mocks/reviews';
+
+type PropsFromRedux = ConnectedProps<typeof connector>
+type ConnectedComponentProps = PropsFromRedux;
 
-function App({cardsAmount, offers, reviews}: AppProps): JSX.Element {
+const mapDispatchToProps = (dispatch: Dispatch<Actions>) => ({
+  offersLoaded(offers: Offer[]) {
```

Comment:
```suggestion
  onOffersLoaded(offers: Offer[]) {
```
пропсы-коллбеки, должны начинаться с `on`